clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -69,11 +69,14 @@
 import org.slf4j.Logger;
 
 import java.net.InetSocketAddress;
+import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.Properties;
+import java.util.TreeSet;
```

Comment:
This is probably unintentional.

Reply:
Thanks much for the quick review. Yeah this is unintentional. I have updated the patch to remove it.